src/Components/PresentationalComponents/Header/Header.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components';
+import Breadcrumb from './Breadcrumb';
+import messages from '../../../Messages';
+import { intl } from '../../../Utilities/IntlProvider';
+
+const Header = ({ title, breadcrumbs, showBreadcrumb, children }) => {
+
+    return (
+        <PageHeader>
+            {showBreadcrumb && <Breadcrumb breadcrumbs={breadcrumbs} />}
+
+            {title && <PageHeaderTitle title={title} />}
+
+            {children}
+        </PageHeader>
+    );
+
+};
+
+Header.defaultProps = {
+    showBreadcrumb: true,
+    title: intl.formatMessage(messages.cvesHeader)
```

Comment:
I think it would be more intuitive if this was empty string by default, because if you want it to display "CVEs" (which is not majority of times) you don't have to pass this prop, but if you want it to display "Reports" you do have to pass it. Maybe not, just an idea. :smile:

Reply:
that's a great point and I totally agree but I was like that previously :( I tried to change it but it breaks a lot of tests which I couldn't resolve :(